ibc/docs/README.md

```diff
@@ -0,0 +1,77 @@
+
+## IBC Module
+
+Pocket's IBC module is split into numerous components detailed below. The overall module layout is as follows:
+
+**Note:** Not all of the different ICS components have been fully implemented yet, this is a work in progress.
+
+```mermaid
+flowchart TB
+    subgraph IBC[IBC Module]
+        subgraph 23[ICS-23]
+            VC[Vector Commitments]
+        end
+        subgraph 24[ICS-24]
+            HR[Host Requirements]
+        end
+        subgraph I[IBC Interface]
+            IF[ICS-25 Handler Interface]
+            IR[ICS-26 Routing Module]
+        end
+    end
+    subgraph 2[ICS-02]
+        CL[Client Semantics]
+    end
+    subgraph 3[ICS-03]
+        CO[Connection Semantics]
+    end
+    subgraph 5[ICS-05]
+        PA[Port Allocation]
+    end
+    subgraph 4[ICS-04]
+        CP[Channel & Packet Semantics]
+    end
+    subgraph 20[ICS-20]
+        FT[Fungible Token Transfer]
+    end
+    IBC ---> 2
+    IBC ---> 3
+    IBC ---> 5
+    IBC ---> 4
+    IBC ---> 20
+```
+
+### Host Configuration
+
+Part of the node configurations relating to the IBC module is as follows:
+
+```json
+"ibc": {
+    "enabled": bool,
+}
+```
+
+If a node enables the IBC module, and is a validator, then the IBC module will automatically create an IBC host on startup. As the host defines the connections, channels and ports - which must stay persistent, the node should be a validator with little risk of suddenly closing any of these while open. As any tokens transferred over a connection/channel/port are unique to that combination they can only be returned over the same combination. If the channel is to close without warning then tokens will be unable to be returned to their source. It is for this reason that only validators are able to become IBC hosts.
+
+## Components
+
+The [IBC specification][ibc-spec] details numerous Interchain Standards (ICSs) that together form the IBC protocol. The following gives an overview of the different components implemented in Pocket's IBC module.
```

Comment:
Are these double square brackets link placeholders? (also "ICS-24" below)

Reply:
The links are at the bottom - I assumed I would be reusing certain ones a few times just makes it easier to read while writing

Reply:
Neat; TIL! I did not not know you could do that :astonished: